### PR TITLE
I couldn't sleep but then I had an idea

### PR DESCRIPTION
**Pull request by jzwar**

### CODE REVIEW

File: src/axis.hpp [hunk 1]
```diff
@@ -134,25 +132,22 @@ inline void Uff(DataType* original_points,          /* in */
...
-  } else {
-    newpointmasks[metric_order_indices[last_index]] = false;
   }
 
   if (stable) {
     IndexingType counter{};
-    stable_inverse.assign(number_of_new_points, -1);
+    // This could be a map (but I think this is sufficient for the moment,
+    // especially if only a few number of duplicates exist)
+    stable_inverse.assign(number_of_points, -1);
```
Comment (j042) [inline src/axis.hpp:150]: doch n_points?

**Review by j042 — APPROVED** [review]
Thanks! I will add arg names so that readme example actually works

File: src/axis.hpp (hunk 1 above)
Comment (jzwar) [reply src/axis.hpp:154]: Actuwlly would be the highest non-duplicate Index, but I didn't want to keep track of yet another value. However, we could use a `std::map` here to save some memory

Comment (j042) [reply src/axis.hpp:154]: Actuwlly I am ok with this

**Pull request merged**